run

```diff
@@ -22,21 +22,34 @@ parser.add_argument(
     action="store_true",
     help="Run container in background & don't show container logs",
 )
+# Set to false if your current user is already in the docker user group
+parser.add_argument(
+    "--no-sudo",
+    default=False,
+    action="store_true",
+    help="Run commands without sudo",
+)
 args = parser.parse_args()
-
 if not os.path.exists("node_modules"):
     print("Node modules have not been installed yet, doing so now")
     subprocess.call(["npm", "install"])
 
 print("Stopping {} container (if running)".format(CONTAINER_NAME))
+remove_container_cmd = ["sudo", "docker", "rm", "--force", CONTAINER_NAME]
+if args.no_sudo:
+    remove_container_cmd.remove("sudo")
```

Comment:
Instead of removing an element how about only adding it if required? That's probably cleaner.

Something like:
```python
cmd = []
if not args.no_sudo:
    cmd.append("sudo")
cmd = ["docker", "rm", "--force", CONTAINER_NAME]
...
```

It would be even cleaner if you reversed the sense of the option: `args.sudo` instead of `args.no_sudo`. Code generally tends to work out cleaner when flags refer to the positive case.

Reply:
Agreed! i was just erring on the side of caution to default to using sudo first.